iife.ts

```diff
@@ -24,8 +26,10 @@ DesignSystemExport;
 ColorPalette;
 ColorToken;
 ColorTokens;
+PreviewForm;
 TypographyTokens;
 DesignSystemPreview;
+FoundationForms;
```

Comment:
Attempting different things here to be able to see the contents of `Form` tab. Still not sure why it's not showing. Any feedback appreciate it.